packages/clerk-js/src/core/clerk.ts

```diff
@@ -159,8 +147,7 @@ export default class Clerk implements ClerkInterface {
   public organization?: OrganizationResource | null;
   public user?: UserResource | null;
   public __internal_country?: string | null;
-  public readonly frontendApi: string;
-  public readonly publishableKey?: string;
+  public readonly publishableKey: string = '';
```

Comment:
The suggestion causes typescript errors (check the screenshot).
Also the current implementation somehow does not make the property readonly (it can be mutated) (see [here](https://www.typescriptlang.org/play?#code/MYGwhgzhAEASCmIQHtoG8BQ1vQA4FcAjEAS2GgCd4wATZAOxAE88BrALmggBcKT6A5tAC80AORiA3Biw5gDHhXzBuyCgApW8Jp0X8BASkw4T0bgAsSEAHS5WI6FqbSTAXwzuM8+j2jmH9PAA7nCIKOpirGIG0hgA9HEAAtwQALTwAB648CrpFBRqGOa29qJiYCjmYGJeCsgg8NYoAurFdgZAA))
So i ended-up to add some tests and use the approach of defining a protected property and a public getter to expose  the `publishableKey` as readonly.

<img width="334" alt="Screenshot 2023-11-09 at 21 13 13" src="https://github.com/clerk/javascript/assets/3936408/988c11ad-8379-4d35-855b-cf2a1ce33104">